src/Compilers/CSharp/Portable/Binder/Binder_Statements.cs

```diff
@@ -193,6 +193,15 @@ private BoundStatement BindFixedStatementParts(FixedStatementSyntax node, Diagno
                                            boundBody);
         }
 
+        private void CheckRequiredLangVersionForAsyncIteratorMethods(DiagnosticBag diagnostics)
+        {
+            var symbol = this.ContainingMemberOrLambda as MethodSymbol;
```

Comment:
Is this certain to succeed?

Reply:
Yes. I'll change to a cast. Thanks

---
In reply to: [223517000](https://github.com/dotnet/roslyn/pull/30280#discussion_r223517000) [](ancestors = 223517000)